portfolio/src/main/java/com/google/sps/servlets/DataServlet.java

```
@@ -77,11 +78,10 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
       datastore.put(taskEntity);
     } else if (maxString != null) {
       try {
-        max = Integer.parseInt(maxString);
-
+        maxNumberOfComments = Integer.parseInt(maxString);
       } catch (NumberFormatException e) {
-        System.out.println("Could not convert to int: " + maxString);
-        max = 1;
+        System.out.println("Could not convert to int|" + maxString+"|");
+        maxNumberOfComments = 1;
```

You might want to return an error code to the caller (probably `SC_BAD_REQUEST`).